### ISSUE TITLE
Error for dependency on PyQT5==5.13

### ISSUE THREAD

**Issue by jbernier040** [CCSI-Toolset/FOQUS#886]
Hello, I have the following installation error:

WARNING: Discarding https://files.pythonhosted.org/packages/fe/f0/927b23d45123d696ae80cd4867c8d955a081ea69dc61b9fbf25e638b54fb/mlrose_hiive-2.1.5.tar.gz#sha256=e45e1bdd7d8638452359528906bfc2e5077deffbdbf1c6c8b1354e03c82ce8e8 (from https://pypi.org/simple/mlrose-hiive/) (requires-python:>=3). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
  Using cached mlrose_hiive-2.1.3.tar.gz (48 kB)
Collecting scipy
  Using cached scipy-1.6.1-cp39-cp39-win_amd64.whl (32.7 MB)
Collecting ccsi-foqus
  Using cached ccsi_foqus-3.5.0-py3-none-any.whl (821 kB)
Collecting mlrose
  Using cached mlrose-1.3.0-py3-none-any.whl (27 kB)
ERROR: Cannot install ccsi-foqus==3.5.0, ccsi-foqus==3.5.1, ccsi-foqus==3.6.0, ccsi-foqus==3.7.0, ccsi-foqus==3.7.1, ccsi-foqus==3.7.2 and ccsi-foqus==3.8.0 because these package versions have conflicting dependencies.

The conflict is caused by:
    ccsi-foqus 3.8.0 depends on PyQt5==5.13
    ccsi-foqus 3.7.2 depends on PyQt5==5.13
    ccsi-foqus 3.7.1 depends on PyQt5==5.13
    ccsi-foqus 3.7.0 depends on PyQt5==5.13
    ccsi-foqus 3.6.0 depends on PyQt5==5.13
    ccsi-foqus 3.5.1 depends on PyQt5==5.13.*
    ccsi-foqus 3.5.0 depends on PyQt5==5.13.*

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

**I installed PyQt5-tools and the PyQt5 version 5.15 was installed, but this cannot fixed installation. How I could fix this?**

**Comment by ksbeattie** [comment]
@jbernier040, can you show me the commands used when getting this error (including the command that installed PyQt5-tools)?

I believe that if you create a separate conda env, just for installing and running FOQUS (with version 5.13 of pyqt5) it should work ok.  That is described in the [install section of our docs](https://foqus.readthedocs.io/en/stable/chapt_install/index.html).

**Comment by jbernier040** [comment]
> @jbernier040, can you show me the commands used when getting this error (including the command that installed PyQt5-tools)?
> 
> I believe that if you create a separate conda env, just for installing and running FOQUS (with version 5.13 of pyqt5) it should work ok. That is described in the [install section of our docs](https://foqus.readthedocs.io/en/stable/chapt_install/index.html).

ksbeattie, I send you the complete process of installation and the errors that appear, I cannot install version 5.13 due to the follow Error: 

How could I  create a separate conda env, just for installing and running FOQUS?
[Anaconda prompt_Install foqus Error.txt](https://github.com/CCSI-Toolset/FOQUS/files/6086510/Anaconda.prompt_Install.foqus.Error.txt)

**Comment by lbianchi-lbl** [comment]
@jbernier040 thanks for trying the Conda installation and reporting the outcome. Can you try again, following this different set of steps?

```sh
conda deactivate
conda env create --force --yes --name ccsi-foqus python=3.7 pip
conda activate ccsi-foqus
python -c 'import sys; print(sys.executable)'
python -m pip install --no-cache-dir ccsi-foqus
python -m pip list
foqus
```

If you could also copy or attach the output of these commands like you did in your last reply, it would be very helpful.

**Comment by jbernier040** [comment]
Ibainchi, thank you for your help, but the same error message appears again (The conflict is caused by:    ccsi-foqus 3.9.0 depends on PyQt5==5.13). I used the set of steps that you send me; however, these did not fix my error of install. I send you an attachment with the output of your commands.

I have installed in mi pc python 3.9.1, should install python 3.7? could this a source of the error of installation of foqus?

best regard
[Anaconda prompt_Install foqus Error_2.txt](https://github.com/CCSI-Toolset/FOQUS/files/6087884/Anaconda.prompt_Install.foqus.Error_2.txt)
,

**Comment by lbianchi-lbl** [comment]
@jbernier040 thanks again for this new check. Unfortunately, it seems that there were a couple of mistakes in the commands I gave you; I'm sorry about that.

Can you try one more time with this fixed version of the previous set of commands?

```sh
conda deactivate
conda env remove --name ccsi-foqus
conda create --name ccsi-foqus --yes python=3.7 pip
conda activate ccsi-foqus
python -c "import sys; print(sys.executable)"
python -m pip install --no-cache-dir ccsi-foqus
python -m pip list
foqus
```

**Comment by jbernier040** [comment]
Ibanchi-lbl, thanks for your help. The installation of foqus was successful. However, the optional software Alamo, is not working correctly. I try to make the example of the surrogate model since foqus, but when I need the optional software Alamo, the following message appears:  

I download and install Alamo in my pc, but I don't know how to fix this error. Could you help me with that?

Best regards,
[Error Alamo optional software_example_usage.txt](https://github.com/CCSI-Toolset/FOQUS/files/6096268/Error.Alamo.optional.software_example_usage.txt)
![Error Alamo excec](https://user-images.githubusercontent.com/79927485/110223972-a8b3ee80-7ea5-11eb-905a-2fb206dbb9e4.png)

**Comment by lbianchi-lbl** [comment]
Hi @jbernier040, happy to hear that the installation worked!

For the ALAMO issue, have you configured the path to the installed ALAMO executable in the "Settings" tab of the FOQUS graphical interface? We realized that the section in our documentation that describes these steps ([link](https://foqus.readthedocs.io/en/stable/chapt_install/install_optional.html)) could be improved, and I've created issue #888 to keep track of it. Feel free to comment on that issue if you have feedback, questions, or suggestions about this.
